MDANSE/Src/MDANSE/Framework/Jobs/DynamicIncoherentStructureFactor.py

```diff
@@ -259,14 +259,13 @@ def finalize(self):
         assign_weights(self._outputData, weight_dict, "s(q,f)_%s")
         for element, number in list(nAtomsPerElement.items()):
             extra_scaling = 1.0 / number
-            self._outputData[f"f(q,t)_{element}"].scaling_factor *= extra_scaling
+            self._outputData[f"f(q,t)_{element}"] *= extra_scaling
```

Comment:
Curious about this being reverted here.

Reply:
This was changed during the NDTSF update but this extra_scaling should have been left in the partial terms so that it has the prefactor 1 / N c_i (where N c_i = N_i). This means that the partial terms are not dependent on the number of atoms.